api/odl.py

```diff
@@ -289,6 +324,11 @@ def checkout(self, patron, pin, licensepool, internal_format):
         if loan.count() > 0:
             raise AlreadyCheckedOut()
 
+        # TODO: Should this look at all loans instead of only loans from the ODL distributor?
```

Comment:
Yes, the discussion in https://github.com/NYPL-Simplified/Simplified/issues/67 makes me think this should be a per-library setting.